docs/source/glossaries.rst

```diff
@@ -95,6 +100,7 @@ UD          Unreliable Datagram (InfiniBand)
 uGNI        user level generic network interface(Cray)
```

Comment:
add whitespace between `interface` and `(Cray)`

Reply:
done